test/test/jfr/JfrTests.java

```diff
@@ -78,14 +79,35 @@ public void parseMultiModeRecording(TestProcess p) throws Exception {
         Assert.isGreater(eventsCount.get("jdk.ObjectAllocationInNewTLAB"), 50);
     }
 
+    private static void waitTtspWarmupOutput(TestProcess p) throws Exception {
+        while (true) {
+            try (FileReader isr = new FileReader(p.getFile(TestProcess.STDOUT));
```

Comment:
Is it necessary to reopen file on each iteration?

Reply:
We could take an `InputStream` from the running process and wait for `read` to return something, but this would result in quite some code (with an additional thread) if we also want to fully support `output = true`. I can do it, I just figured we could keep it simple since it's test code